src/screens/BenefitManipulationScreen.tsx

```diff
@@ -73,7 +76,6 @@ export default function BenefitManipulationScreen({ navigation, Benefit }) {
               name='desc'
               isInvalid={Boolean(errors.desc)}
             />
-            {/* todo: style text */}
```

Comment:
moje komentarze 

![1067860950700531842](https://github.com/StampWallet/stamp-wallet/assets/93001484/1f92a5b3-1e66-462d-970d-510393af3d7b)

Reply:
Uznałem, że ostylowanie tekstu jest gotowe, revert?

Reply:
nieee to tylko tzw. zart